src/ngraph/op/roi_align.cpp

```diff
@@ -0,0 +1,193 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "roi_align.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+constexpr NodeTypeInfo op::v3::ROIAlign::type_info;
+
+op::v3::ROIAlign::ROIAlign(const Output<Node>& input,
+                           const Output<Node>& rois,
+                           const Output<Node>& batch_indices,
+                           const size_t pooled_h,
```

Comment:
According to our opset3::ROIAlign spec these attributes are integer.

Reply:
I believe that this is not critical. We do not strictly specify data type as 'int' or 'size_t' in the spec for attribute values. Since the supported values for these attributes are positive integer values then 'size_t' is fine.

Reply:
I've changed it to int. WE've always used size_t when we knew that attribute values should be positive. Please let me know which version should be the final one.